docs/user/aws/install_upi.md

```diff
@@ -114,6 +114,14 @@ open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 
 If you do so, you'll need to [add ingress DNS records manually](#add-the-ingress-dns-records) later on.
 
+#### Disconnected clusters
+
+For disconnected clusters, Openshift has to be configured not to manage DNS, otherwise [the ingress operator][ingress-operator] will try to contact the STS endpoint "sts.amazon.com" directly  as opposed to the configured VPC endpoint for the cluster. Use the following command to configure Openshift not to manage DNS:
+
+```sh
+oc patch dnses.config.openshift.io/cluster --type=merge --patch='{"spec":{"privateZone":null,"publicZone":null}}'
```

Comment:
`oc patch` would work in a running cluster, but it is best if users handle this pre-install, the steps for which are in the previous section.

Reply:
Fixed.